src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__header {
+    margin-bottom: 16px;
+  }
+
+  // .body-card__header
+  // .body-card__title
+  &__title {
+    margin: 0;
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 1.5;
+    margin-bottom: 4px;
+    display: block;
+  }
+
+  // .body-card__product-code
+
+  &__product-code {
+    margin: 0;
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+  }
+
+  // .body-card__stars
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  // .body-card__block-price
+
+  &__block-price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  // .body-card__text
+
+  &__text {
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 1.5;
+  }
+
+  // .body-card__price
+
+  &__price {
+    color: #060b35;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 1.13;
+  }
+
+  // .body-card__button
+
+  &__button {
+    height: 40px;
+    width: 100%;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+    background-color: #00acdc;
+    font-weight: 700;
+    color: #fff;
+    font-size: 14px;
+    line-height: 1.14;
+    text-transform: uppercase;
+  }
+
+  &__button:hover {
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    background-color: #fff;
+    color: #00acdc;
+  }
+}
```

Comment:
Consider using SCSS variables for main values such as colors, font sizes, and dimensions. This will make it easier to maintain and update styles across your application.